packages/event-status-service/src/controllers/event-status/create.ts

```diff
@@ -0,0 +1,62 @@
+import { Response } from 'express'
+import { logger } from '../../core/logger'
+import { AppRequest, stringifyError } from '@reapit/node-utils'
+import { FunctionExpression, AttributePath } from '@aws/dynamodb-expressions'
+import { db } from '../../core/db'
+import { generateStatusItem } from '../../schemas/event-status.schema'
+
+type payload = {
```

Comment:
It's a tiny thing and just code taste but I tend to favour PascalCase for interfaces and types - just so they standout from variables in the code